### PR TITLE
Kozmaz87/simpleconsumer

### PR DESCRIPTION
**Pull request by kozmaz87**
Contains boilerplate for twisted and the node architecture base classes.
Simple producer. Timed operation mode. (missing the folder export feature)
Simple consumer. Logs received documents. (the time resolution algorithm is yet to be implemented)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,11 +69,28 @@ The root schemadocument is called *ebutt_live.xsd*.
 The Python library
 ==================
 
-The library uses XSD schemas from the xsd1.1 subdirectory. The bindings are auto-generated by PyXB.
-For binding generation one can use the following code:
-
-    pyxbgen -r --binding-root=./ebu_tt_live/bindings -m __init__ --schema-root=./ebu_tt_live/xsd1.1/ -u ebutt_live.xsd
```
Comment (nigelmegitt) [inline README.md]: Why remove the bindings generation instructions?

File: ebu_tt_live/clocks/base.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        hours, minutes, seconds, microseconds = self._get_time_members(checked_time)
+        # We have our most significant value. Time for range check
+        if hours > 23:
+            raise TimeFormatOverflowError(ERR_TIME_FORMAT_OVERFLOW)
+        centiseconds, _ = divmod(microseconds, 10000)
+        return '{hours:02d}:{minutes:02d}:{seconds:02d}:{centiseconds:02d}'.format(
+            hours=hours,
+            minutes=minutes,
+            seconds=seconds,
+            centiseconds=centiseconds
```
Comment (nigelmegitt) [inline ebu_tt_live/clocks/base.py]: This needs to be frame times not centiseconds, for smpte time expressions.

File: README.md (hunk 1 above)
Comment (kozmaz87) [inline README.md]: because the makefile has the line. Also couple lines above that the same line is present.

Comment (nigelmegitt) [inline README.md]: Sorry, my mistake - that's the danger of only looking at the diffs.

**Comment by nigelmegitt** [comment]
Given that we've raised spec [issue 115](https://git.ebu.io/ebutt/ebutt-part-3/issues/115), I'm happy to merge this now with the smpte flaws present, since we may as a result of the issue remove the smpte timebase parts. It's not worth spending extra time on them now until we have a resolution to that spec issue.

**Comment by nigelmegitt** [comment]
I'm going to hold off actually merging for a little while though in case there's a chance of getting a 2nd review, which would be great.

**Comment by kozmaz87** [comment]
I came to the decision that I close this pull request and merge it with the consumerlogic branch That builds on top of this one. 

I am making architectural changes according to the code review

**Pull request closed without merging**